third_party/terraform/tests/resource_billing_budget_test.go

```diff
@@ -0,0 +1,68 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccBillingBudget_billingBudgetCurrencycode(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"billing_acct":  getTestBillingAccountFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
+		},
+		CheckDestroy: testAccCheckBillingBudgetDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBillingBudget_billingBudgetCurrencycode(context),
+				Check: resource.ComposeTestCheckFunc(
+					// currency_code is returned when it is not included in the request
+					resource.TestCheckResourceAttrSet("google_billing_budget.budget", "amount.0.specified_amount.0.currency_code"),
```

Comment:
This doesn't really need to be here. The test will detect if there is a diff between the config after it is applied, so if the currency_code field is not computed it will see a diff with this config